CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- Added support for retrieving the `Punctuated` sequence of parameters in a `FunctionBody`
```

Comment:
Put this under ### Added.

Reply:
Not resolved--this needs to be under ### Added, and not ### Changed 😛

Reply:
Whoops!